<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenuWindow</class>
 <widget class="QMainWindow" name="MainMenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainMenu</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>:/Logos/output-onlinepngtools (2).png</selectedon>
   </iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Logo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>231</width>
      <height>201</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/output-onlinepngtools (2).png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="LogoName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Brush Script MT</family>
      <pointsize>28</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; color:#00009e;&quot;&gt;Anti Fraud Damage Detector&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LocalizationButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Classification</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TrainButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Train System</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>332</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Logos.qrc"/>
 </resources>
 <connections/>
</ui>
